### DISCUSSION TITLE
todo.txt - option to disable creation date of task

### DISCUSSION THREAD

**Discussion by naunan** [gsantner/markor#1778] [Ideas] [answered]
It would be great to have an option to disable the automatic date (creation date) when adding a new task.

**Comment by gsantner** [comment, marked as answer]
There is already an option. When you have the file open::  
Click the file settings at the top menu, there disable "Auto-Format".

![grafik](https://user-images.githubusercontent.com/6735650/182046332-16922910-f661-45b7-ae7d-eb509f03c3b9.png)

**Reply by naunan** [reply]
Thanks, that totally works. I thought the option was about the lines and colored labels/projects, but maybe I've should have tried it. :)

**Reply by gsantner** [reply]
> about the lines and colored labels/projects

This is "Syntax highlighting", the option is on the same page as well!
